subsys/bootloader/Kconfig

```diff
@@ -139,6 +99,13 @@ config SB_DISABLE_NEXT_W
 	  It uses RRAMC's region 4 and is limited to 31KB for nRF54L15, nRF54L10 and nRF54L05
 	  and to 127KB for nRF54LV10a and nRF54LM20a.
 
+config SB_MONOTONIC_COUNTER_ROLLBACK_PROTECTION
+	bool "HW monotonic version counter rollback protection"
+	help
+	  Enable this option to ensure that the secure bootloader prevents
```

Comment:
reword to say the option signifies if it has been enabled by sysbuild, not that it should be enabled here by a user.